libpod/container_exec.go

```diff
@@ -954,18 +954,22 @@ func (c *Container) removeAllExecSessions() error {
 	}
 	// Delete all exec sessions
 	if err := c.runtime.state.RemoveContainerExecSessions(c); err != nil {
-		if lastErr != nil {
-			logrus.Errorf("Error stopping container %s exec sessions: %v", c.ID(), lastErr)
+		if errors.Cause(err) != define.ErrCtrRemoved {
```

Comment:
Should also check for `ErrNoSuchCtr`.

I wonder if we need a `IsCtrRemoved` function somewhere (`libpod/define` maybe?) to make this common check easy.

Reply:
I don't think we can hit `ErrNoSuchCtr` since we're operating on this specific container already.

At least, I couldn't manage to hit this error.  Are you sure we can hit it there?

> I wonder if we need a IsCtrRemoved function somewhere (libpod/define maybe?) to make this common check easy.

Yes, I think it would be nice consolidate such state checks.

Reply:
I think you may be right, since we're at most doing update calls on an existing container we probably can't trigger a `ErrNoSuchCtr`.